HW4/04.1 Practical task_Yuliia Holovii.py

```diff
@@ -0,0 +1,6 @@
+Celsius = float(input("Enter the temperature in Celsius: "))
+if Celsius < -273.15:
+    print("Error: Temperature below absolute zero (-273.15°C)")
+else:
+    Fahrenheit = (Celsius * 9/5) + 32
+print (f"{Celsius}°C is equivalent to {Fahrenheit}°F")
```

Comment:
This print statement should be inside the else block, not outside it. In the current implementation, it will always print (f"{Celsius}°C is equivalent to {Fahrenheit}°F"), even for invalid values